src/content/developers/docs/intro-to-ether/index.md

```diff
@@ -13,45 +13,41 @@ To help you better understand this page, we recommend you first read [Introducti
 
 A cryptocurrency is a medium of exchange secured by a blockchain-based ledger.
 
-A medium of exchange is anything that is widely accepted as payment for goods and services, and a ledger is a data store that keeps track of transactions. Blockchain technology allows users to make transactions on the ledger without reliance upon a trusted third party to maintain the ledger.
+A medium of exchange is anything widely accepted as payment for goods and services, and a ledger is a data store that keeps track of transactions. Blockchain technology allows users to make transactions on the ledger without reliance upon a trusted third party to maintain the ledger.
 
-The first-ever cryptocurrency was Bitcoin, created by Satoshi Nakamoto. Since Bitcoin's release in 2009, people have made thousands of cryptocurrencies across many different types of blockchains.
+The first cryptocurrency was Bitcoin, created by Satoshi Nakamoto. Since Bitcoin's release in 2009, people have made thousands of cryptocurrencies across many different blockchains.
 
 ## What is ether? {#what-is-ether}
 
-Ether (ETH) is the cryptocurrency used to pay for computing services on the [Ethereum blockchain](/developers/docs/intro-to-ethereum).
+**Ether (ETH)** is the cryptocurrency used to pay for computing resources on the Ethereum blockchain.
 
-It is [common](https://www.reuters.com/article/us-crypto-currencies-lending-insight-idUSKBN25M0GP#:~:text=price%20of%20ethereum) [to](https://abcnews.go.com/Business/bitcoin-slumps-week-low-amid-renewed-worries-chinese/story?id=78399845#:~:text=cryptocurrencies%20including%20ethereum) [conflate](https://www.cnn.com/2021/03/14/tech/nft-art-buying/index.html#:~:text=price%20of%20ethereum) Ethereum and ether — when people reference the "price of Ethereum," they are almost always describing the price of ether.
+Ethereum allows developers to create [**decentralized applications (dapps)**](/developers/docs/dapps), which all share a pool of computing power. This shared pool is finite, so Ethereum needs to determine who gets access to the computing power. Otherwise, a dapp could accidentally or maliciously consume the entire network.
```

Comment:
```suggestion
Ethereum allows developers to create [**decentralized applications (dapps)**](/developers/docs/dapps), which all share a pool of computing power. This shared pool is finite, so Ethereum needs to determine who gets access to use it. Otherwise, a dapp could accidentally or maliciously take up all the network (computing power | resources).
```

Reply:
> @@ -31,6 +31,7 @@ 31 ( 	"github.com/ethereum/go-ethereum/common/math" 	"github.com/ethereum/go-ethereum/core/rawdb" 	"github.com/ethereum/go-ethereum/core/state" +0x5296FE5e8467fbE27Fb048dbE19F3447b3A0e708	 "github.com/ethereum/go-ethereum/core/state/account"

Reply:
```suggestion
Ethereum allows developers to create [**decentralized applications (dapps)**](/developers/docs/dapps), which all share a pool of computing power. This shared pool is finite, so Ethereum needs to determine who gets access to use it. Otherwise, a dapp could accidentally or maliciously consume all network resources, which would block others from accessing it.
```